### PR TITLE
simplified css-importing syntax.

### PR DESCRIPTION
**Pull request by luciuschoi**
simplified @import summernote syntax and refactored the view template with unobtrusive javascript pattern. 

So. You can import the summernote css in application.scss as follows:

```
...
@import "summernote";
...
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,34 +26,39 @@ $ bundle install
...
 ```html
 <div id="summernote">Hello Summernote</div>
+```
 
-<script type="text/javascript">
-  $(document).ready(function() {
-    $('#summernote').summernote();
-  });
-</script>
+In app/assets/javascripts/summernote_bootstraped.coffee, you should add the following:
```
Comment (seapy) [inline README.md]: i think document should mention about both javascript and coffeescript

File: README.md [hunk 2]
```diff
@@ -86,41 +93,47 @@ and update summernote option
...
+In app/assets/javascripts/summernote_bootstraped.coffee, you should add as follows:
 
-<script type="text/javascript">
-  $(document).ready(function() {
-    $('#summernote').summernote({
-      lang: 'ko-KR'
-    });
-  });
-</script>
+```coffee
```
Comment (seapy) [inline README.md]: i think document should mention about both javascript and coffeescript

File: README.md [hunk 3]
```diff
@@ -86,41 +93,47 @@ and update summernote option
...
 ```html
 <div id="summernote">Hello Summernote</div>
+```
+
+In app/assets/javascripts/summernote_bootstraped.coffee, you should add as follows:
 
-<script type="text/javascript">
-  $(document).ready(function() {
-    $('#summernote').summernote({
+```coffee
```
Comment (seapy) [inline README.md]: i think document should mention about both javascript and coffeescript

**Comment by luciuschoi** [comment]
Do you mean the usage of both should be described in README.md file ?

**Comment by seapy** [comment]
yes :)

**Comment by luciuschoi** [comment]
I've just committed the updated README.md file. ^^
You can find it in the "Commits" tab on the top of this page.

**Comment by luciuschoi** [comment]
Maybe, you should bump up to v0.6.6.1 because the  file structure of `stylesheets/` directory was changed. Please keep in mind that you should rake release the updated version to rubygems.org.

**Pull request merged**